portal/views/auth.py

```diff
@@ -360,7 +402,7 @@ def base64_url_decode(s):
         """url safe base64 decoding method"""
         padding_factor = (4 - len(s) % 4)
         s += "=" * padding_factor
-        return base64.b64decode(unicode(s).translate(
+        return base64.b64decode(s.translate(
```

Comment:
not related to this PR - just a broken line of code left over from the python2 days.